### PR TITLE
Remove guest_accelerator

### PR DESCRIPTION
**Pull request by marko7460**

### CODE REVIEW

File: examples/node_pool/main.tf [hunk 1]
```diff
@@ -57,16 +58,17 @@ module "gke" {
...
-      accelerator_type  = "nvidia-tesla-p4"
       image_type        = "COS"
       auto_repair       = false
       service_account   = "${var.compute_engine_service_account}"
     },
   ]
 
   node_pools_oauth_scopes = {
-    all = []
+    all = [
```
Comment (morgante) [inline examples/node_pool/main.tf]: These look like unrelated changes. Can you rebase on master?

Comment (marko7460) [reply examples/node_pool/main.tf]: i can, but the example keeps recreating `node_pools` on every converge without this being set.

**Comment by chrislovecnm** [comment]
@marko7460 why did we remove this??

**Comment by morgante** [comment]
We removed it due to an [upstream issue](https://github.com/GoogleCloudPlatform/magic-modules/pull/1924).

Now that the upstream issue is fixed, we can add it back in. Do you mind submitting a PR?

**Comment by marko7460** [comment]
i can add it back. Ill have a PR ready soon
